pipelines/azure-pipelines.load-test.yml

```diff
@@ -13,13 +12,23 @@ variables:
 
 steps:
 
-- script: |
-    az login --service-principal --username $(ARM_CLIENT_ID) --password $(ARM_CLIENT_SECRET) --tenant $(ARM_TENANT_ID)
-    az account set --subscription $(ARM_SUBSCRIPTION_ID)
-  displayName: 'SETUP: Login on Azure CLI'
+- task: AzureKeyVault@1
+  inputs:
+    azureSubscription: $(TF_VAR_SERVICE_CONNECTION_NAME)
+    KeyVaultName: $(TF_VAR_KEY_VAULT_NAME)
+    SecretsFilter: 'acr-secret,arm-client-secret'
+  displayName: 'Get secrets from Azure Key Vault'
+
+- task: AzureCLI@2
+  displayName: 'Login to Azure Container Registry'
+  inputs:
+    azureSubscription: $(TF_VAR_SERVICE_CONNECTION_NAME)
+    scriptType: bash
+    scriptLocation: inlineScript
+    inlineScript: |
+      az acr login -n $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME) -u $(TF_VAR_JMETER_IMAGE_REGISTRY_USERNAME) -p $(acr-secret)
```

Comment:
How about merging the following `docker pull $(TF_VAR_JMETER_DOCKER_IMAGE)` command on this same `AzureCLI` task? I saw you separated them, but if they represent the same task (validate the JMeter docker image) they should be together. Do you agree?

Reply:
Also, we don't need to specify the username and the secret. So this would become:
```shell
az acr login -n $(TF_VAR_JMETER_IMAGE_REGISTRY_NAME)
docker pull $(TF_VAR_JMETER_DOCKER_IMAGE)
```

Reply:
done